Makefile

```diff
@@ -35,6 +35,10 @@ test-linux:
 		swift:5.7-focal \
 		bash -c 'apt-get update && apt-get -y install make && make test-swift'
 
+build-for-static-stdlib:
+	@swift build -c debug -Xswiftc -static-stdlib
+	@swift build -c release -Xswiftc -static-stdlib
```

Comment:
note that I've been using `swift build --static-swift-stdlib` (double `-`) - not sure if that's the same as `-Xswiftc -static-stdlib`